pkg/languagedetection/cli_detector.go

```diff
@@ -9,9 +9,15 @@ import (
 	"regexp"
 	"strings"
 
+	"github.com/DataDog/datadog-agent/pkg/languagedetection/internal/detectors"
 	"github.com/DataDog/datadog-agent/pkg/languagedetection/languagemodels"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
+var cliDetectors = []languagemodels.Detector{
+	&detectors.JRubyDetector{},
```

Comment:
Nit: Should this be a pointer? The recievers that implement languagemodels.Detector don't take a pointer type.

Reply:
Nice catch, I missed this one.